a1/README.md

```diff
@@ -3268,6 +3269,34 @@ Use [Gulp](http://gulpjs.com) or [Grunt](http://gruntjs.com) for creating automa
 
 **[Back to top](#table-of-contents)**
 
+## Low Level Functions
+
+Angular 1.x ships with some [helper functions](https://docs.angularjs.org/api/ng/function) that are found within JavaScript natively. In such cases we should favor usage of native JavaScript functions instead of Angular's helper functions. In other cases where functionality is not found in JavaScript natively, we should turn to utility libraries such as [lodash](https://lodash.com/docs) to fill-in the gaps. Here are some examples:
+
+|Replace | With | |
+|---|---|---|
+|`angular.isArray()` | `Array.isArray()` | |
+|`angular.forEach()` | `Array.prototype.forEach()` | _for arrays_ |
+|`angular.forEach()` | `_.forEach()` | _for objects_ |
+|`angular.isUndefined()` | `_.isUndefined()` | |
+|`angular.copy()` | `_.cloneDeep()` |  |
+|`angular.extend()` | `Object.assign()` | _available in ES2015+_ |
+
+
+###### [Style [Y500](#style-y500)]
+
+  - Avoid using Angular low-level functions such as `angular.copy()`, `angular.extend()` or even `angular.isUndefined()`.
+
+  *Why?*: You can replace a lot Angular's low-level functions with native ES2015+ functionality such as [`Object.assign()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) for copying objects and the [Spread Syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator#Copy_an_array) for copying arrays _(ie. `[...arr1]`)_.
+
+  *Why?*: This will ease the process of upgrading to Angular 2 as it does not ship with these helper functions.
```

Comment:
others why's ... (needs wordsmithing) ...

- Because it's baked into ES. They were written at a time when they were not readily availableas they are today, thus there is no need for a framework specific wrapper. 

- These ES features are already more commonly used, as they are ES native.